nifi-manifest/nifi-runtime-manifest-core/src/main/java/org/apache/nifi/runtime/manifest/impl/StandardComponentManifestBuilder.java

```diff
@@ -33,6 +35,8 @@ public class StandardComponentManifestBuilder implements ComponentManifestBuilde
     private final List<ProcessorDefinition> processors = new ArrayList<>();
     private final List<ControllerServiceDefinition> controllerServices = new ArrayList<>();
     private final List<ReportingTaskDefinition> reportingTasks = new ArrayList<>();
+    private final List<ParameterProviderDefinition> parameterProviders = new ArrayList<>();
+    private final List<FlowAnalysisRuleDefinition> flowAnalysisRules = new ArrayList<>();
```

Comment:
Just a heads up that the same two lists need to be added to the actual `ComponentManifest` class, and this builder needs to set them:

https://github.com/apache/nifi/blob/main/c2/c2-protocol/c2-protocol-component-api/src/main/java/org/apache/nifi/c2/protocol/component/api/ComponentManifest.java#L32

Currently the final manifest.json doesn't end up with the parameter providers and flow analysis rules.

Reply:
Good catch @bbende - pushed a commit to add this